### ISSUE TITLE
Update overlay hover popovers

### ISSUE THREAD

**Issue by Irev-Dev** [KittyCAD/modeling-app#2390]
When hovering on the overlays users see this:
<img width="322" alt="image" src="https://github.com/KittyCAD/modeling-app/assets/29681384/4195f889-ac8d-444f-83a7-566834178d49">

But designs have since been updated eg:

<img width="353" alt="image" src="https://github.com/KittyCAD/modeling-app/assets/29681384/89c95c9e-dd5d-44c7-aca3-f60b22386fef">

[See all designs here](https://www.figma.com/file/5UUPOawPLEs3D0Gptf42vy/Designs?type=design&node-id=1612-8663&mode=design&t=xBc2dorvFnwaIh2l-0)

**Comment by jessfraz** [comment]
is it weird i kinda like the first one better?

**Comment by Irev-Dev** [comment]
Oh CC @franknoirot 

I know you said "hard to put into words" so I'm guessing you just mean the vibe of it and I guess the composure?

One of the things I liked about Frank's design (second one) is he's able to emphasise "Unconstrained", without it being the first thing in the popover.

**Comment by jessfraz** [comment]
yes I like that i think something about the font size, i cant really describe, just feels different

**Comment by franknoirot** [comment]
Maybe it feels a little top-heavy because there's a bunch of big text relative to how much smaller text there is. And I think the container could be feeling a bit cramped. Here's what I tweaked in this version:
1. Remove the "[Function Name] ·" prefix, because it adds a lot of that larger font size. I'll have a think about how to communicate what line function is being used.
2. moved the Constrained/Unconstrained tag up to the top, bumped down its padding. I think it sharing a row with that description text feels a little more cramped too.
3. Double the padding around the whole card

![Screenshot 2024-05-20 at 12 04 16 PM](https://github.com/KittyCAD/modeling-app/assets/23481541/df067153-18d2-4d43-8542-6c8828fd5cd1)
![Screenshot 2024-05-20 at 12 04 16 PM](https://github.com/KittyCAD/modeling-app/assets/23481541/df067153-18d2-4d43-8542-6c8828fd5cd1)

**Comment by Irev-Dev** [comment]
This feels less cramped.

In regards to 1. we are still highlighting in the editor

![2024-05-21 14 45 25](https://github.com/KittyCAD/modeling-app/assets/29681384/94162a89-bfff-4c86-af59-a52c52faccce)

But is the thinking about when the editor is closed?

**Comment by jessfraz** [comment]
ooo i like those better
